08/passenger-service/src/test/java/cz/muni/fi/airportmanager/passengerservice/resource/PassengerResourceTestIT.java

```diff
@@ -0,0 +1,146 @@
+package cz.muni.fi.airportmanager.passengerservice.resource;
+
+import cz.muni.fi.airportmanager.passengerservice.entity.Passenger;
+import cz.muni.fi.airportmanager.passengerservice.model.CreatePassengerDto;
+import io.quarkus.test.common.http.TestHTTPEndpoint;
+import io.quarkus.test.junit.QuarkusIntegrationTest;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Collections;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.is;
+
+@QuarkusIntegrationTest
+@TestHTTPEndpoint(PassengerResource.class)
+class PassengerResourceTestIT {
```

Comment:
Why this isn't in the PassengerResourceTest?

Reply:
Tests are focused on the whole service without any isolation